### PR TITLE
feat!: allow null fuel signal values

### PR DESCRIPTION
**Pull request by hhovsepi**
These changes allow the `range`, `percentRemaining`, and `amountRemaining` attributes to be set to null. This wasn't possible with primitive double types, as they always have a default value (0.0) if not explicitly set.

Asana: https://app.asana.com/0/1205850229375854/1205850230294701/f

### CODE REVIEW

File: src/test/java/com/smartcar/sdk/VehicleTest.java [hunk 1]
```diff
@@ -169,9 +169,9 @@ public void testFuel() throws Exception {
 
     VehicleFuel fuel = this.subject.fuel();
 
-    Assert.assertEquals(fuel.getAmountRemaining(), 53.2);
-    Assert.assertEquals(fuel.getPercentRemaining(), 0.3);
-    Assert.assertEquals(fuel.getRange(), 40.5);
+    Assert.assertEquals(fuel.getAmountRemaining(), Double.valueOf(53.2));
+    Assert.assertEquals(fuel.getPercentRemaining(), Double.valueOf(0.3));
+    Assert.assertEquals(fuel.getRange(), Double.valueOf(40.5));
```
Comment (wikiadrian) [inline src/test/java/com/smartcar/sdk/VehicleTest.java:174]: I am wondering if we could add a test that gets at least one signal with null?

**Review by wikiadrian — APPROVED** [review]
I validated manually the SDK 

LGTM!!
